frontera/settings/default_settings.py

```diff
@@ -50,6 +51,7 @@
 }
 SQLALCHEMYBACKEND_REVISIT_INTERVAL = timedelta(days=1)
 STATE_CACHE_SIZE = 1000000
+STATE_CACHE_SIZE_LIMIT = 0
```

Comment:
I still don't see why state_cache_size_limit option was introduced and where it is used?